opm/parser/share/keywords/000_Eclipse100/C/COMPDAT

```diff
@@ -6,7 +6,7 @@
     {"name" : "K2" , "value_type" : "INT" },
     {"name" : "STATE" , "value_type" : "STRING" , "default" : "OPEN"},
     {"name" : "SAT_TABLE" , "value_type" : "INT" , "default" : 0},
-    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*LiquidVolume/Time*Pressure"},
+    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*ReservoirVolume/Time*Pressure"},
```

Comment:
@andlaus sums it up quite nicely; there is an extremely basic parser which understands '*' and *one* '/': https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/Units/UnitSystem.cpp#L99